upgrade/tests/upgrade_package/test_check_upgrade_from_local_wheel.py

```diff
@@ -132,3 +134,61 @@ def test_upgrade_top_level_package_from_local_wheel_where_package_name_does_not_
     expected = "Failed to install wheel"
     actual = out
     assert expected in actual
+
+
+def test_upgrade_local_wheel_test_constraints_flag(
+    wheels_dir, use_pip, mocked_constraints_path
```

Comment:
question: do we need `mocked_constraints_path` in these two tests? 
suggestion: if not, please remove